projects/arlas-components/src/lib/components/results/model/item.ts

```diff
@@ -114,6 +114,14 @@ export class Item {
    * @description color characterising the item
    */
   public color: string;
+  /**
+    * @description Title for the details section.
+    */
+  public detailsTitle: string;
+  /**
+   * @description If details item title is enabled
+   */
+  public detailTitleEnabled: boolean;
```

Comment:
In one property you wrote detail**s** and the other just detail. Could it be uniformized ?

Reply:
Fixed